server/protocols/webadmin/webadmin-mailrepository/src/main/java/org/apache/james/webadmin/service/ClearMailRepositoryTask.java

```diff
@@ -91,6 +97,7 @@ public long getRemainingSize() {
         return mailRepositories
                 .stream()
                 .map(Throwing.function(MailRepository::size).sneakyThrow())
-                .reduce(0L, (a, b) -> a + b);
+                .mapToLong(Long::valueOf)
```

Comment:
can't you apply mapToLong to `MailRepository::size` directly ?

Reply:
It's a `long`, not a `Long`, the typing is too strong :p